elitzur-core/src/main/scala/com/spotify/elitzur/validators/Implicits.scala

```diff
@@ -37,10 +37,10 @@ trait Implicits {
   implicit def wrappedValidator[T: Validator]: Validator[ValidationStatus[T]] = new WrappedValidator[T]
   implicit def optionValidator[T: Validator]: Validator[Option[T]] = new OptionValidator[T]
   implicit def dynamicTypeValidator[T <: DynamicValidationType[_, _, _]: ClassTag]: DynamicValidator[T] = new DynamicValidator[T]
-  implicit def seqValidator[T: Validator: ClassTag]: Validator[Seq[T]] = wrapSeqLikeValidator(() => Seq.newBuilder[T])
-  implicit def listValidator[T: Validator: ClassTag]: Validator[List[T]] = wrapSeqLikeValidator(() => List.newBuilder[T])
-  implicit def arrayValidator[T: Validator: ClassTag]: Validator[Array[T]] = wrapSeqLikeValidator(() => Array.newBuilder[T])
-  implicit def vectorValidator[T: Validator: ClassTag]: Validator[Vector[T]] = wrapSeqLikeValidator(() => Vector.newBuilder[T])
+  implicit def seqValidator[T: Validator: ClassTag]: Validator[Seq[T]] = wrapSeqLikeValidator( "Seq", () => Seq.newBuilder[T])
```

Comment:
Related to treating all of these the same, IMO the strings should be removed

Reply:
Yeah makes sense on this, and with the validation loop performance.

Reply:
I'll revert that stuff